pages/developers/guidelines.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Package requirements and guidelines
+permalink: /guidelines
+nav_order: 1
+parent: Developer information
+---
+
+# Package requirements and guidelines
+
+The below provides requirements and guidelines for Scikit-HEP packages,
+which we hope help providing high-quality packages with some level of homogeneity across them.
```

Comment:
I would add a line like: New packages should adhere to these minimum requirements. Further optional suggestions are in the following pages." or something like that.

Reply:
Good point. Added.